src/Compilers/Core/Portable/Emit/EditAndContinue/EmitBaseline.cs

```diff
@@ -359,7 +379,7 @@ public static EmitBaseline CreateInitialBaseline(ModuleMetadata module, Func<Met
             Debug.Assert((ordinal == 0) == (encId == default));
             Debug.Assert((ordinal == 0) == (initialBaseline == null));
             Debug.Assert((ordinal == 0) == (anonymousTypeMap == null));
-            Debug.Assert((ordinal == 0) == (synthesizedDelegates == null));
+            Debug.Assert((ordinal == 0) == (anonymousDelegates == null));
```

Comment:
Nit: Should there be a corresponding assert here for `anonymousDelegatesWithFixedTypes`?